ytplayer.py

```diff
@@ -75,12 +91,83 @@ async def _resume(self,ctx: commands.Context):
         
         self.voice.resume()
 
+    @commands.command(name='skip')
+    async def _skip(self,ctx: commands.Context):
+        """Skip a song"""
+        
+        self.voice.stop()
+
     @commands.command(name='stop')
     async def _stop(self,ctx: commands.Context):
-        """Stop audio playpack"""
+        """Stop audio playback"""
         
+        self.queue = []
         self.voice.stop()
 
+    @commands.command(name='move')
+    async def _move(self,ctx: commands.Context, old_pos, new_pos):
+        """Change postition of a song in queue"""
+        
+        queue_len = len(self.queue)
+        if queue_len == 0:
+            await ctx.send("what do you want to move you dingus")
+        else:
+            try:
+                old_pos = int(old_pos) - 1
+                new_pos = int(new_pos) - 1
+                queue_range = range(0, queue_len)
+                if old_pos in queue_range and new_pos in queue_range:
+                    self.queue.insert(new_pos, self.queue.pop(old_pos))
+                else:
+                    await ctx.send("youh numbhas ar not in tha queue")
+            except ValueError:
+                await ctx.send("you idiot itsa not a numbha")
+    
+    @commands.command(name='clear')
+    async def _clear(self,ctx: commands.Context):
+        """Empty queue"""
+        
+        self.queue = []
+    
+    @commands.command(name='queue')
+    async def _queue(self,ctx: commands.Context):
+        """Show queue"""
+
+        if len(self.queue) == 0:
+            await ctx.send("Queue is empty you dingus")
+        else:
+            number = 1
+            string = ""
+            for x in self.queue:
+                string = string + str(number) + ". " + x.title + "\n"
+                number = number + 1
+            await ctx.send(string)

```

Comment:
1) Don't we want to keep it in embed like the rest?
2) Suggestion: Maybe we should print only a few first entries for queue and it's length?